android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/Connection.java

```diff
@@ -30,7 +31,7 @@ public static InputStream getInputStream() {
         return urlConnection.getInputStream();
       }
     } catch (IOException e) {
-      Log.e("Connection", "getInputStream failed: " + e);
+      KMLog.LogException(TAG, "getInputStream failed: ", e);
```

Comment:
How is this compiling? Where is it getting `KMLog` from?

Reply:
`Connection` and `DownloadIntentService` are in the same `com.tavultesoft.kmea.util` package as `KMLog`.